### DISCUSSION TITLE
Slack app integration

### DISCUSSION THREAD

**Discussion by uoy-david-whitehead** [community/community#24776] [Apps, API and Webhooks]
<p>The latest Github Slack app <a href="http://my.slack.com/apps/A01BP7R4KNY-github" rel="noopener nofollow ugc">my.slack.com/apps/A01BP7R4KNY-github</a> is failing to integrate with channels in workspaces on our org after the update last week. In channels with the app installed we get “/github failed with the error “app_not_installed_on_team””. The new app has been approved at the org level in Slack, is there anything we need to change on the github side of things?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Would this issue be the same as what you’re experiencing?</p>
<aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/integrations/slack/issues/1154" target="_blank" rel="noopener">github.com/integrations/slack</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/integrations/slack/issues/1154" target="_blank" rel="noopener">Even after installation, new bot in private channels reports `app_not_installed_on_team`</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-04-15" data-time="13:29:09" data-timezone="UTC">01:29PM - 15 Apr 21 UTC</span>
      </div>

        <div class="date">
          closed <span class="discourse-local-date" data-format="ll" data-date="2021-04-15" data-time="16:12:27" data-timezone="UTC">04:12PM - 15 Apr 21 UTC</span>
        </div>

      <div class="user">
        <a href="https://github.com/bethac07" target="_blank" rel="noopener">
          <img alt="bethac07" src="https://user-images.githubusercontent.com/11161530/181086649-68d19e93-2f0e-4a6d-9d88-c4ee399eb9cf.jpeg" class="onebox-avatar-inline" width="20" height="20">
          bethac07
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">We had several GitHub-bot private channels get the upgrade message this morning, we invited the new bot, but now /github messages...</p>
</div>

<div class="labels">
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by uoy-david-whitehead** [comment]
<p>Hi canuckjacq,</p>
<p>That’s exactly the issue, sorry for not finding that post when I was searching yesterday. I can see that there is still a workspace in our Org that is using the Github (Legacy) app. I’ll ask those users to update their integration.</p>
<p>Thank you for the link and solution! Sorry for the duplicate post.</p>

**Comment by canuckjacq** [comment]
<p>No problem at all! I bet more people will find the solution here because you asked.</p>
